templates/remove-credential-confirmation

```diff
@@ -56,6 +55,10 @@ $(document).ready(function () {
   }
 });
 
+function cancelRemoval() {
+  window.location.replace("/login/mfa/manage?mfa-state=" + mfaState);
```

Comment:
What does this do if the candid is running as a subpath? as in https://api.jujucharms.com/identity

Reply:
changed.. the url is now passed to the template..